src/main/java/ServerPort.java

```diff
@@ -0,0 +1,37 @@
+package com.td.HTTPServer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Date;
+
+public class ServerPort {
+
+  ServerSocket server;
+  Socket clientSocket;
+
+  public ServerPort(int portNumber) throws IOException {
+    server = new ServerSocket(portNumber);
+  }
+
+  public String getRequest() throws IOException {
```

Comment:
nitpick: `getRequest` might be a little confusing in the context of HTTP (where a GET request is a specific thing)